SwitchSpeak-iOS/SwitchSpeak-iOS/CrumbManager.swift

```diff
@@ -71,7 +84,17 @@ class CrumbStack {
     func push(string:String) {
         let newCrumb = Crumb(content: string)
         items.append(newCrumb)
-        newCrumb.setLocationAndColor(byIndex: items.count - 1)
+        if items.count > 4 {
```

Comment:
Why are you checking if items.count > 4? Please explain in a comment or make a static variable (like above) so that it's clear.